server/src/config/config.js

```diff
@@ -0,0 +1,8 @@
+/* eslint import/no-dynamic-require: 0 */
+const config = require(`./config.${process.env.NODE_ENV}`);
+
+export const spreadsheetID = () => (config.default.spreadsheetID);
```

Comment:
Why the need to export each config variable separately and as function? How about just importing `config` where they are used, then use as `config.spreadsheetID`